client/verta/tests/test_endpoint/test_endpoint.py

```diff
@@ -292,7 +292,7 @@ def test_form_update_body(self):
                                   'resources': {'cpu_millis': 500, 'memory': '500Mi'}, 'strategy': 'rollout'}
 
 
-    def test_get_deployed_model(self, client, experiment_run, model_for_deployment, created_endpoints):
+    def test_get_deployed_model(self, client, experiment_run, model_version, model_for_deployment, created_endpoints):
```

Comment:
This test is kinda checking two things now: We should mention that for future devs!
```suggestion
    def test_get_deployed_model(self, client, experiment_run, model_version, model_for_deployment, created_endpoints):
    """
    Verifies prediction for a finished deployment, as well as for an endpoint
    in the middle of being updated.

    """
```

Reply:
Added.